core/src/main/java/org/apache/calcite/util/Bug.java

```diff
@@ -220,6 +220,12 @@ public abstract class Bug {
    * is fixed. */
   public static final boolean CALCITE_6328_FIXED = false;
 
+  /** Whether
```

Comment:
The title of the commit that introduces this change is misleading: you are using the convention used for a commit which *solves* an issue, but this commit only works around the issue. So I think before merging the title of the commit should be changed.

Reply:
Do you have any suggestions for a commit message, I think it might be good to include this in [CALCITE-6586] commit.

Reply:
I have no problem with 6586, just with 6328

Reply:
The commit message can be "Introduced bug guard for CALCITE-6328."

Reply:
Done.